src/Stacktrace.php

```diff
@@ -311,7 +311,7 @@ protected function getFrameArgumentsValues(array $frame): array
             $result = array_map([$this, 'serializeArgument'], $frame['args']);
         } else {
             foreach (array_values($frame['args']) as $index => $argument) {
-                $result['param' . ($index + 1)] = $this->serializeArgument($argument);
+                $result['param' . ((int)$index + 1)] = $this->serializeArgument($argument);
```

Comment:
So, the culprit of the problem as we found out is XDebug. The error occurs because it changes how the arguments of a stacktrace frame are reported by using the argument name as key of each value of the `$args` variable instead of its 0-based position in the function signature. Blindly casting `$index` to an `integer` then would result in some indesiderate results (e.g. `foo` would be converted to `0`). The best fix we can do is to check whether the `$index` variable is a `string` or an `integer` and act accordingly. However, this exact same issue should have been fixed in #761 which is the reason why there is an `if`/`else` statement wrapping the loop, so I don't get why it happens and it would be cool if you could debug a bit more in depth to see what's happening